cip-TOKEN-STD/cip-meiersi-Canton-Network-Token-Standard.md

```diff
@@ -0,0 +1,567 @@
+<pre>
+  CIP:  CIP XXXX
+  Layer: Daml
+  Title: Canton Network Token Standard
+  Author: Simon Meier, Matteo Limberto, Moritz Kiefer
```

Comment:
@moritzkiefer-da @matteolimberto-da: might I ask you for a review of this first draft?

Your call on whether you'd like to be listed as an author or not.

Reply:
> Your call on whether you'd like to be listed as an author or not, I'd be happy to have you as a co-author.

don't have an issue with being listed although it feels like I'm stealing your credit given that you've done all the hard work on this CIP. Your choice whether you wanna keep me in the list or not.

Reply:
I consider the author list here to be the main contributors to the design of the change. You've earned your place here.